app/components/footerlinks/footerlinks.js

```diff
@@ -58,30 +64,42 @@ const FooterLinks = (props) => {
           target="_blank"
         >Get Mobile App</a>
       </div>
+    }
       <div className='footer-link large-format-only'>
         <a
-          href="http://support.tidepool.org/"
+          href={__BRANDING__ !== 'diabeloop' ? 'http://support.tidepool.org/' : URL_DIABELOOP_SUPPORT}
```

Comment:
can we put the links in variable that we set first depending on the BRANDING? and then reuse those variables. 
it should make the code more readable

Reply:
I added a constant for display links based on \_\_BRANDING\_\_ constant